### ISSUE TITLE
Basic Tutorial Needed

### ISSUE THREAD

**Issue by w4nderlust** [haifengl/smile#2]
The project seems supercool and well documented, but I think it needs a basic tutorial to get started and get to know the main classes to use. Something as simple as:

1. import the data
2. split in train and test set
3. train a classifier on the training set
4. test the classifier on the test set

**Comment by Xyclade** [comment]
A good example to start with would be a short tutorial how to classify males/females ([example from wikipedia](http://en.wikipedia.org/wiki/Naive_Bayes_classifier) )

**Comment by haifengl** [comment]
A basic tutorial is now available at https://github.com/haifengl/smile/wiki/Tutorial:-A-Gentle-Introduction-of-SmileMiner

Please let me if you have any questions and suggestions. I will keep adding more wiki pages. Thanks!

**Comment by w4nderlust** [comment]
Very nice, thank you!

**Comment by haifengl** [comment]
Thanks for quick response. Sorry for the long wait. Not productive during the holiday season :)
